src/data/content/modelli/contribuisci-ai-modelli.yaml

```diff
@@ -0,0 +1,512 @@
+              align: "middle"
+              color: "secondary"
+            label: Modelli #C
+            url: "../" #M
+      otherInfo:
+        items:
+          - title: Rispettando
+            tooltip: "Norme e riferimenti che i contenuti di questa pagina ti aiutano a rispettare"
+            icon:
+              icon: "sprites.svg#it-pa"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Linee guida di design #C
+            url: "/norme-e-riferimenti/linee-guida-di-design/" #M
+      personalInfo:
+         items: #C
+          - title: Sei
+            tooltip: "Se non ti riconosci in questi profili non preoccuparti: potrai scoprire informazioni utili per saperne di più. In alternativa, naviga la sezione superiore per approfondire il contesto"
+            icon:
+              icon: "sprites.svg#it-user"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Designer, Developer #C
+          - title: Vuoi
+            tooltip: "Obiettivi di progettazione per cui sono utili i contenuti di questa pagina"
+            icon:
+              icon: "sprites.svg#it-flag"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: "Progettare e sviluppare l'interfaccia utente" #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Modelli,Interfaccia utente,Sviluppo interfaccia]
+
+  sectionsEditorial:
+
+    - noSpace: false
+      centered: true
+      title: "L'evoluzione dei modelli di sito e servizi per la Pubblica Amministrazione è aperta"
+      #text: Scopri come contribuire in base alle tue competenze e al tempo che hai a disposizione
+      background: light
+      components:
+        - name: TextImageCta
+          # title: Un approccio aperto e trasparente
+          noSpace: true
+          text: |  
+            Le risorse di Designers Italia sono mantenute con un approccio aperto e trasparente. Crediamo che il contributo attivo della community che osserva, utilizza, personalizza e implementa quotidianamente le soluzioni proposte sia fondamentale per migliorarne la qualità. 
+            
+          # ctas: 
+          #   - label: "Conosci il fondamento Princìpi e intenzioni?" #C
+          #     # screenReaderText: ""
+          #     url: "/design-system/fondamenti/principi-e-intenzioni/" #M
+          #     blank: false #M #C true if new tab for external links
+          #     icon:
+          #       icon: sprites.svg#it-arrow-right #C #I #it-arrow-right | it-external-link for external links
+          #       color: primary
+          #       align: middle
+          #       hidden: true
+          #       size: sm
+          #       addonClasses: ms-2
+            
+    - centered: true
+      noSpace: false
+      title: Come contribuire ai modelli
```

Comment:
Da qui in poi va spostato a un livello inferiore tecnico. 

Un blocco card con una card sola che porti alla parte tecnica. Una scheda che si potrebbe chiamare "[Modello di contribuzione](https://designers.italia.it/design-system/come-contribuire/modello-di-contribuzione/)" per coerenza con il design system volendo. Oppure anche "Contribuisci ai modelli" volendo lasciare l'azione.